### DISCUSSION TITLE
Can't run Blazor wasm startproject

### DISCUSSION THREAD

**Discussion by VervaekeSofie** [dotnet/aspnetcore#49649] [Q&A]
I'm new with blazor wasm.
I created a project with Visual Studio Community and build it.
Then I want to Run it, but that went wrong. I tried it with Google chrome and Edge, the same error is generated.
The error is: Error: Failed to start platform. Reason: TypeError: Failed to fetch
    at At (https://localhost:7292/_framework/blazor.webassembly.js:1:59617) {stack: 'Error: Failed to start platform. Reason: Type…292/_framework/blazor.webassembly.js:1:59617)', message: 'Failed to start platform. Reason: TypeError: Failed to fetch'}

![image](https://github.com/dotnet/aspnetcore/assets/84907679/cc1b0954-7525-49a8-8400-0d402da076c3)
![image](https://github.com/dotnet/aspnetcore/assets/84907679/5cccfe78-8838-4e17-9bd3-6fa20ad33d87)

I tried it also with Visual Code and the same happend. I work with Windows 11. Is there something not installed?
If I make a Blazor Server app, I can run it.

**Comment by LeaFrock** [comment]
Looks like a NotFound problem. Check the `index.html` first.

For example, if you're hosting the wasm app behind a gateway, you need to change `<base href="/" />` to `<base href="/xxx/" />` so that all js/css files are accessable.

**Reply by VervaekeSofie** [reply]
Do you mean like this?
![image](https://github.com/dotnet/aspnetcore/assets/84907679/89146126-3d56-42aa-b209-f1e0468b9b48)
of like this?
![image](https://github.com/dotnet/aspnetcore/assets/84907679/dddf66fb-479d-458c-bc35-154cbb899376)

Both changes gives me the same problem

**Reply by LeaFrock** [reply]
No... I just put up an example. There would be different kinds of reasons on your problem. 

I suggest you upload and share a mini repo so that others can do help.
